portfolio/src/main/webapp/fetchHello.js

```diff
@@ -0,0 +1,21 @@
+async function showText() {
+    try{
+        const responseFromServer = await fetch('/hello');
+        const res = await responseFromServer.json();
+        
+        const textContainer = document.getElementById('showText');
+        
+        const messages = [`My favorite song is ${res.song}`,
+                        `My favorite movie is ${res.movie}`,
+                        `My favorite series is ${res.series}`, 
+                        `My favorite sport is ${res.sport}`, 
+                        `My favorite game is ${res.game}`,                
+    ];
+
+        const randNum = Math.floor((Math.random()* (messages.length + 1)));
```

Comment:
This looks like it returns a random number from 0 to 5 (both included), which could result in an out-of-bounds error since messages[5] doesn't exist. Based on this w3schools [page](https://www.w3schools.com/js/js_random.asp), if we remove the `+ 1`, that should make it 0 to 4 instead.